source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc

```diff
@@ -0,0 +1,20 @@
+[[Configuring_timezones]]
+= Configuring the time zone
+
+{virt-product-fullname} stores time zone configurations for virtual machines in */etc/ovirt-engine/conf/00-timezone.properties*. This file contains default time zone values such as `Etc/GMT=Greenwich Standard Time`. It features mappings that are valid for windows and non-Windows time zones.
```

Comment:
"Windows and non-Windows"

Reply:
@stoobie From what I understand, there are subtle differences between Windows and non-Windows time zones. For example: Windows typically uses the local timezone for the machine's internal time, so it needs timezone files which can translate back to UTC. Linux typically uses UTC as the machine's internal time, so it needs timezone files which can translate to local time.

Reply:
Gotcha. Thanks for the explanation.